librubyfmt/src/ruby.rs

```diff
@@ -50,13 +51,13 @@ pub const Qnil: VALUE = VALUE(8);
 
 extern "C" {
     // stuff that we need to compile out rubyfmt
-    pub fn ruby_init();
+    pub fn ruby_setup() -> libc::c_int;
     pub fn ruby_cleanup(_: libc::c_int);
     pub fn rb_eval_string_protect(_: *const libc::c_char, _: *mut libc::c_int) -> VALUE;
     pub fn rb_funcall(_: VALUE, _: ID, _: libc::c_int, ...) -> VALUE;
     pub fn rb_utf8_str_new(_: *const libc::c_char, _: libc::c_long) -> VALUE;
     pub fn rb_str_new_cstr(_: *const libc::c_char) -> VALUE;
-    pub fn rb_string_value_cstr(_: VALUE) -> *const libc::c_char;
+    pub fn rb_string_value_cstr(_: *const VALUE) -> *const libc::c_char;
```

Comment:
I don't see a corresponding change to the definition. Are you sure this is correct?

Reply:
https://github.com/ruby/ruby/blob/7bafc9f433223d82aac8e8974b9bfdbdc34ca4df/include/ruby/internal/core/rstring.h#L92